### PR TITLE
add lap

### PR DESCRIPTION
**Pull request by PrimozGodec**
Adding because of https://github.com/conda-forge/staged-recipes/pull/4737

### CODE REVIEW

File: recipes/lap/meta.yaml [hunk 1]
```diff
@@ -20,10 +20,9 @@ requirements:
   build:
     - python
     - setuptools
-
+    -numpy      >=1.10.1
   run:
     - python
-    - numpy     >=1.10.1
     - cyton     >=0.21
```
Comment (astaric) [inline recipes/lap/meta.yaml]: cython? And it is probably build requirement or not a requirement at all?

File: recipes/hyper/mete.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% set name = "lap" %}
```
Comment (astaric) [inline recipes/hyper/mete.yaml]: Is this file supposed to be here?

File: recipes/lap/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  imports:
+    - lap
+
+about:
+  home: https://github.com/gatagat/lap
+  license: BSD-2-Clause
+  summary: 'Linear Assignment Problem solver (LAPJV/LAPMOD).'
+  license_family: BSD
```
Comment (dougalsutherland) [inline recipes/lap/meta.yaml]: Also add `license_file: LICENSE`, please.

File: recipes/lap/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [py2k]
```
Comment (dougalsutherland) [inline recipes/lap/meta.yaml]: Homepage says it works for 2.7; is there a reason you're skipping it?

Comment (PrimozGodec) [reply recipes/lap/meta.yaml]: Sorry, my mistake, fixed

**Comment by dougalsutherland** [comment]
AppVeyor also missed this one; toggling closed/open to trigger it.

**Comment by dougalsutherland** [comment]
Whoops, I missed earlier that:
- this package needs `toolchain`
- it also needs to [handle numpy requirements](https://conda-forge.org/docs/meta.html#building-against-numpy) specially, because it compiles against it
- `scipy` doesn't seem to actually be a requirement

I went ahead and made those changes for you; CI will be done in the morning and hopefully I can merge then. :)

**Comment by dougalsutherland** [comment]
Oh, one other comment: it would be nice to add a real test that the package actually works, since we're compiling code and it's conceivable that would go wrong. It ships with some `pytest` tests; not sure if those take too long and we should just run a subset.

**Comment by dougalsutherland** [comment]
Anyway, since you have other things that depend on this I'm going to merge now (CI finished, forget about that whole "morning" thing...) and open an issue on the feedstock to add real tests later.

Thanks @PrimozGodec!

**Comment by PrimozGodec** [comment]
Thank you @dougalsutherland

**Pull request merged**